website/docs/rbs-support.md

```diff
@@ -242,6 +242,8 @@ during type checking:
 | [Tuple type]           | `[Foo, Bar]`                             | [`[Foo, Bar]`](tuples.md)                                                                                            |
 | [Shape type]           | `{ a: Foo, b: Bar }`                     | [`{ a: Foo, b: Bar }`](shapes.md)                                                                                    |
 | [Proc type]            | `^(Foo) -> Bar`                          | [`T.proc.params(arg: Foo).returns(Bar)`](procs.md)                                                                   |
+| [Block type]           | `{ (Foo) -> Bar }`                       | [`T.proc.params(arg: Foo).returns(Bar)`](procs.md)                                                                   |
+| [Optional Block type]  | `?{ (Foo) -> Bar }`                      | [`T.nilable(T.proc.params(arg: Foo).returns(Bar))`](procs.md#optional-blocks)                                        |
```

Comment:
I originally had these as a single row with `<br>`, but that didn't look great.

This looks better, although the optional block line ends up split over two lines due to the way the table gets formatted. I might try opening **a different PR** that switches the last column to use actual <code>```ruby</code> code blocks, and spread the `type_parameters` example over multiple lines, which I suspect would fix that and read better anyway (on top of giving us formatting for the Sorbet code).

Reply:
I've opened https://github.com/sorbet/sorbet/pull/8831 to change the table/snippet formatting

Reply:
#8831 is being merged, so I'll rebase ahead of it to streamline review.

Reply:
☝️ fyi. @jez